packit_service/worker/helpers/build/build_helper.py

```diff
@@ -754,3 +754,21 @@ def report_status_to_configured_job(
             links_to_external_services=links_to_external_services,
             update_feedback_time=update_feedback_time,
         )
+
+    def notify_about_failure_if_configured(self):
+        """
+        If there is a failure_comment_message configured for the job,
+        post a comment and include the configured message. Do not post
+        the comment if the last comment from the Packit user is identical.
+        """
+        if (
+            configured_message := self.job_config.notifications.failure_comment_message
+        ) is None:
+            return
+
+        formatted_message = configured_message.format(
+            commit_sha=self.db_project_event.commit_sha
```

Comment:
I quite like the job_name variable, or, maybe some links might be requested. But we can freely stay with this and add on request later on.

Reply:
I removed it since users can define their custom messages per-job which can replace this, but I am not strongly against

maybe @martinpitt could help us with the feedback here:  would you be interested in Packit providing some other variables besides `commit_sha`  (see [the docs in preparation](https://github.com/packit/packit.dev/blob/6cc46346bbce77ad013594631422f2b57a779fee/docs/configuration/index.md#failure_comment)) that you could utilise in the configuration of the custom failure messages? :)

Reply:
The big enabler here is to get a configurable list of people to ping, all the details can then be examined on the PR page. 

Personally I liked the first version (i.e. just the first commit) a bit better: it's (mildly) useful to have the job name in the message, and it is IMHO much better to *not* interpolate `configured_message` -- adding variables to the text sounds like a recipe for bugs.

But it's not a strong opinion -- I'd be happy either way (and really looking forward to this!)

Reply:
thanks for the feedback! I can see that as @lachmanfrantisek wrote before, some users would like to have this completely customizable and by providing the commit_sha, actually, the duplication of the comment in the PR can be configured as well: when {commit_sha} is not present in the configured message, there will be only one Packit comment per PR and when it will be present, Packit will place a new comment for each failure for different commit

I would probably test it in staging in the current form and we can still easily tweak the content of the comment if needed